music21/environment.py

```diff
@@ -1111,6 +1111,7 @@ def xmlReaderType(self):
         Returns an xmlReaderType depending on the 'musicxmlPath'
 
         >>> a = environment.Environment()
+        >>> original = a['musicxmlPath']  #_DOCS_HIDE
```

Comment:
this should be okay as is, since environment.Environment() does not write directly to disk unless `.save()` is called -- environment.UserSettings() does that.  (Why these two different versions is a long and boring old story.  Maybe they should be merged at some point.  But the notion was that people might want to temporarily change environment for different runs)

Reply:
That makes sense, but something strange is going on, then, because I only did this to get the tests to pass on Ubuntu during the period I was actually trying to run everything through MuseScore.

I just tried it again, and lo & behold -- failures if I revert this change (and try to open musescore):

compare [a run without this change](https://github.com/jacobtylerwalls/music21/pull/34/checks?check_run_id=2865340468)

to [restoring the change](https://github.com/jacobtylerwalls/music21/pull/34/checks?check_run_id=2865371614)

Except for the unrelated failures at the top of the log owing to my lack of effort to install a PNG reader, notice the failures all happen (reverse) alphabetically after the environment module, and we have code opening musescore well before that.

Do you want to debug what's going on or just merge it?

Reply:
> Do you want to debug what's going on or just merge it?

Sorry I didn't see this immediately, I see now we're talking about two different things; you've correctly pointed out no change is necessary to avoid disturbing a developer's local system, but you can also see how without this change we spoil test isolation and any later unit tests run in the same session as this doc test will choke on nonexistent filepaths. So I think we need to keep this change.